UPGRADING.md

```diff
@@ -1,6 +1,39 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.2.5
+
+[Virtus](https://github.com/solnic/virtus) isn't used for coercing anymore. If your project depends on Virtus, you have to add it to your `Gemfile`. Also, if Virtus is used as a custom type
```

Comment:
Simlify and shorten, "is no longer used for coercing parameters" and just ", add it to your Gemfile"